ES6_modules/index.js

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable import/prefer-default-export */
+
+import Book from './modules/Book.js';
+import { getBook, addBook, removeBook } from './modules/Storage.js';
+
+document.addEventListener('DOMContentLoaded', () => {
+  const buttonAdd = document.querySelector('.addbook');
+  const bookList = document.querySelector('.books');
+
+  // Function to create a book element
+  function createBookElement(book) {
+    const bookElement = document.createElement('div');
+    bookElement.classList.add('bookstyle');
+    bookElement.innerHTML = `
+      <div>
+        <p>${book.title}</p>
+        <p>${book.author}</p>
+        <button class="remove-btn">remove</button>
+      </div>
+    `;
+    return bookElement;
+  }
+
+  // Function to render all books
+  function renderBooks() {
+    const books = getBook();
+    bookList.innerHTML = '';
+    books.forEach((book) => {
+      const bookElement = createBookElement(book);
+      bookList.appendChild(bookElement);
+    });
+  }
```

Comment:
- [ ] Great job on implementing the changes in your project! However, as per the project requirements, it's necessary to use arrow syntax for functions. Could you please review your index file and the modules, and refactor all the functions to use ES6 syntax?
Arrow functions are a concise way to write functions in JavaScript. Here's an example of how to convert a regular function to arrow syntax:

```js
// Regular function
function myFunction(param) {
  // function body
}

// Arrow function
const myFunction = (param) => {
  // function body
}
```
Please update all the functions in your codebase accordingly, using arrow syntax. If you need any further assistance or have specific questions about the refactoring process, feel free to ask!